### PR TITLE
Use openSUSE Leap 15 Rocky Horizon image

### PR DESCRIPTION
**Pull request by jgu17**
Use openSUSE Leap 15 Rocky Horizon image, and required Horizon and
helm toolkit chart version.

### CODE REVIEW

File: site/soc/software/config/versions.yaml [hunk 1]
```diff
@@ -8,6 +8,13 @@ data:
         location: /armada/airship-components/openstack-helm-infra
         subpath: ingress
     osh:
+      # TODO remove after Treasuremap catches up on uplifting version
+      horizon:
+        location: https://git.openstack.org/openstack/openstack-helm
+        #reference: 38fb90d52824f9d23c99aab91e23097cd9b657d9
```
Comment (kantarun) [inline site/soc/software/config/versions.yaml]: Should we remove that commented version ?

Comment (jgu17) [reply site/soc/software/config/versions.yaml]: yes :-)

**Comment by jgu17** [comment]
> LGTM.
> Do we need to carry these overrides here ..is not there is proposal to add values_overrides directory and then having suse specific overrides there. Need to check may be its not merged yet.

not sure what you referring to. maybe post a link?  This is the same overrides osh deployment uses now.

**Comment by jsuchome** [comment]
Yes, OSH is currently using these overrides.
Eventually we want to move them upstream, see https://review.openstack.org/#/c/651491/ but it may take some time.

**Comment by jsuchome** [comment]
The change looks good, but why does the PR have 3 commits? Can you squash them into one and correctly rebase against master?

File: site/soc/software/config/versions.yaml (hunk 1 above)
Comment (jgu17) [reply site/soc/software/config/versions.yaml]: removed.

**Pull request merged**